package NDnnProto;


message TStdpState {
    optional double Y = 1;
    repeated double X = 2;
    repeated double FirstMoment = 3;
    repeated double SecondMoment = 4;
}

message TStdpConst {
    optional double TauPlus = 1;
    optional double TauMinus = 2;
    optional double Aplus = 3;
    optional double Aminus = 4;
    optional double TauFirstMoment = 5;
    optional double TauSecondMoment = 6;
}

